contracts/modules/viewer/NexusViewer.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import {Multicall} from "../../abstract/Multicall.sol";
+import {IAssessment} from "../../interfaces/IAssessment.sol";
+import {IAssessmentViewer} from "../../interfaces/IAssessmentViewer.sol";
+import {IGovernance} from "../../interfaces/IGovernance.sol";
+import {INXMMaster} from "../../interfaces/INXMMaster.sol";
+import {IPooledStaking} from "../../interfaces/IPooledStaking.sol";
+import {IStakingViewer} from "../../interfaces/IStakingViewer.sol";
+import {ITokenController} from "../../interfaces/ITokenController.sol";
+
+/// @title NexusViewer Contract
+/// @notice This contract provides a unified view of system-wide data from various contracts within the Nexus Mutual platform.
+contract NexusViewer is Multicall {
+  struct LegacyPoolStake {
+    uint deposit;
+    uint reward;
+  }
+
+  struct ClaimableNxm {
```

Comment:
this struct is missing assessmentStake, in case there is stake that's not locked

Reply:
would be good to leave a comment on each row with what it represents - i.e. 
```
    uint governanceRewards;  // Governance rewards
    IStakingViewer.AggregatedTokens aggregateStakingTokens; // Expired stake deposits + staking rewards
    IAssessmentViewer.AssessmentRewards assessmentRewards; // Assessment rewards
    LegacyPoolStake legacyPooledStake; // Legacy pooled staking deposits + rewards
    uint v1CoverNotesAmount; // Legacy cover note deposits
```

Reply:
consider splitting `IStakingViewer.AggregatedTokens aggregateStakingTokens; // Expired stake deposits + staking rewards` into 2: `expiredStake` and `rewards`, as only these should be considered when calculating the claimable nxm. this way, you can abstract away some of this cognitive overhead from the FE developers